components/Layout/Layout.tsx

```diff
@@ -13,7 +12,7 @@ import Header from 'components/Header/Header'
 import Footer from 'components/Footer/Footer'
 import PageNotFound from 'components/PageNotFound'
 import SEOHead from 'components/SEO'
-import Newsletter from '../Newsletters';
+import Newsletter from '../Newsletters'
```

Comment:
```suggestion
import Newsletter from 'components/Newsletters'
```

begitu bisa tydac ngab?